aiosip/pytest_plugin.py

```diff
@@ -0,0 +1,133 @@
+import asyncio
+import contextlib
+import gc
+import sys
+
+import pytest
+
+
+try:
+    import uvloop
+except:  # pragma: no cover
```

Comment:
I'm not certain what error `uvloop` could raise on import but it might be best to only catch `ImportError` ?

Reply:
Yeah, good point. Copy and pasted blindly from aiohttp.

We can do a few things better in here in general since its now possible to write hooks and implement coroutine based fixtures as well with newer pytest releases.

Reply:
Fixed